### ISSUE TITLE
Clarification on the levels of encryption using dns66

### ISSUE THREAD

**Issue by chrisr666** [julian-klode/dns66#333]
Hi Julian, sorry for posting this as an issue when it's not but I couldn't see a way to directly message you.
Can you clarify what level of encryption dns66 vpn applies? Is all traffic encrypted end to end?
Thanks

**Comment by tijnludger** [comment]
Hello, I noticed no one has responded to your issue. DNS66 is not a VPN. It only uses the VPN API in Android so it can have access to your DNS queries. Anything that is not a DNS query will not be touched and or read. Everything is done locally on your device, no data is uploaded, your device does the blocking.

Basically when your phone makes a DNS request it will first be read by DNS66, if the DNS query is known to be used to serve ads or trackers it will be replied to with a blank response, essentially blocking it from connecting to the advertiser. Again, none of this is done in the "cloud", it is all done locally so encryption methods are not really applicable to this app.

**Comment by chrisr666** [comment]
Hi, ah right, in that case can I use dns66 on top of a VPN?




Get Outlook for Android







On Tue, May 28, 2019 at 3:03 PM +0100, "tijnludger" <notifications@github.com> wrote:












Hello, I noticed no one has responded to your issue. DNS66 is not a VPN. It only uses the VPN API in Android so it can have access to your DNS queries. Anything that is not a DNS query will not be touched and or read. Everything is done locally on your device, no data is uploaded, your device does the blocking.


Basically when your phone makes a DNS request it will first be read by DNS66, if the DNS query is known to be used to serve ads or trackers it will be replied to with a blank response, essentially blocking it from connecting to the advertiser. Again, none of this is done in the "cloud", it is all done locally so encryption methods are not really applicable to this app.

**Comment by julian-klode** [comment]
DNS66 does not do any encryption and it's not possible to combine it with a VPN.
